.github/workflows/zeebe-ci.yml

```diff
@@ -157,10 +157,13 @@ jobs:
         uses: ./.github/actions/observe-build-status
         with:
           job_name: "integration-tests/${{ matrix.group }}"
-          build_status: ${{ contains(steps.*.conclusion, 'failure') && 'failure' || 'success' }}
+          build_status: ${{ job.status }}
```

Comment:
:thought_balloon: Haven't double checked if that works (i.e. that `job.status` is "failure" if any of the previous steps failed) but I'd trust you on this

Reply:
Oh yes that is a copy&pasta from the Unified CI [ci.yml](https://github.com/camunda/camunda/blob/main/.github/workflows/ci.yml#L89) where I introduced it to test what would happen (when I created it) and it has proven to work 🙂